checkstyle/import-control-core.xml

```diff
@@ -65,7 +65,6 @@
     <allow pkg="kafka.api"/>
     <allow pkg="kafka.server"/>
     <allow pkg="org.apache.kafka.clients.admin"/>
-    <allow pkg="integration.kafka.server" class="IntegrationTestHelper"/>
```

Comment:
It seems the class `IntegrationTestHelper` was moved due to review comment (#9986)

this should be included by separate PR and the `core/src/test/java/kafka/test/junit/README.md` need to be updated as well.


@mumrah FYI

Reply:
Ack. Opened a new PR #10126.